package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "smart-guides",
+  "name": "snapsterts",
   "version": "0.0.0",
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "tsc && vite build",
```

Comment:
While Vite does transpile TS for us out the box ( pretty much ) it doesn't do type checking ... so this is added by Vite to enable type checking on the build as another "safety net", i do agree that if we are type checking during development this might not be needed but something we can discuss :)